src/__internal__/utils/logger/index.ts

```diff
@@ -1,30 +1,31 @@
 /* eslint-disable no-console */
-
-// Globally enable the logger
-let enabled = process.env.NODE_ENV !== "production";
+import getNodeEnv from "./get-node-env";
 
 /**
  * Utility for dispatching messages to the browser console.
  * By default, logging is disabled in production mode.
  */
 const Logger = {
-  setEnabledState: (newState: boolean) => {
```

Comment:
praise: good idea to remove I don't think I've seen it used outside of the unit tests